sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md

```diff
@@ -2,13 +2,9 @@
 
 ## 2.24.0-beta.1 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
 ### Bugs Fixed
 
-### Other Changes
+- Fixed a bug that updating `NetworkSecurityGroup` rules' source/destination fails when from application security group to ip addresses/service tags.
```

Comment:
source/destination of `NetworkSecurityGroup` rules ?

Reply:
Changed, thanks!